### ISSUE TITLE
Can't  convergence when trained with batch 128 by using the softmax loss

### ISSUE THREAD

**Issue by globallocal** [BVLC/caffe#5137]
Hello, everyone, I was so confused about the following misconvergence. 
**Have you ever encounter the problem ?
And could you give me some advices ?**
Thank you so much.

****
I1229 23:55:59.560392 27153 solver.cpp:229] **Iteration 2300, loss = 5.07946**
I1229 23:55:59.560902 27153 solver.cpp:245]     Train net output #0: accuracy = 0.25
I1229 23:55:59.560936 27153 solver.cpp:245]     Train net output #1: softmax_loss = 5.07946 (* 1 = 5.07946 loss)
I1229 23:55:59.560951 27153 sgd_solver.cpp:106] Iteration 2300, lr = 0.1
I1230 00:00:51.582455 27153 solver.cpp:229] **Iteration 2400, loss = 4.88092**
I1230 00:00:51.585647 27153 solver.cpp:245]     Train net output #0: accuracy = 0.28125
I1230 00:00:51.585752 27153 solver.cpp:245]     Train net output #1: softmax_loss = 4.88092 (* 1 = 4.88092 loss)
I1230 00:00:51.585778 27153 sgd_solver.cpp:106] Iteration 2400, lr = 0.1
I1230 00:05:42.707515 27153 solver.cpp:229] **Iteration 2500, loss = 87.3365**
I1230 00:05:42.708058 27153 solver.cpp:245]     Train net output #0: accuracy = 0
I1230 00:05:42.708091 27153 solver.cpp:245]     Train net output #1: softmax_loss = 87.3365 (* 1 = 87.3365 loss)
I1230 00:05:42.708112 27153 sgd_solver.cpp:106] Iteration 2500, lr = 0.1
I1230 00:10:33.261914 27153 solver.cpp:229] **Iteration 2600, loss = 87.3365**
I1230 00:10:33.262436 27153 solver.cpp:245]     Train net output #0: accuracy = 0
I1230 00:10:33.262531 27153 solver.cpp:245]     Train net output #1: softmax_loss = 87.3365 (* 1 = 87.3365 loss)

**Comment by naibaf7** [comment]
User questions should go into the google group, this is not a bug.
On another note, use a lower learning rate, like 0.01 or 0.001.

**Comment by globallocal** [comment]
Thank you so much, I will check it now...
